lib/document.js

```diff
@@ -10,36 +9,10 @@ export default ({ head, css, html, data, dev, staticMarkup, cdn }) => {
     </head>
     <body>
       <div id='__next' dangerouslySetInnerHTML={{ __html: html }} />
-      {staticMarkup ? null : <script dangerouslySetInnerHTML={{ __html: '__NEXT_DATA__ = ' + htmlescape(data) }} />}
-      {staticMarkup ? null : createClientScript({ dev, cdn })}
+      {staticMarkup ? null : <script dangerouslySetInnerHTML={{
+        __html: `__NEXT_DATA__ =${htmlescape(data)}; module={};`
+      }} />}
+      <script type='text/javascript' src='/main.js' />
     </body>
   </html>
 }
-
-function createClientScript ({ dev, cdn }) {
```

Comment:
btw, the reasons of `next.js` client are to cache it as a chunk, and CDN makes the cache more efficient by sharing the same file across sites.